packages/angular/cli/bin/ng

```diff
@@ -3,9 +3,9 @@
 
 // Provide a title to the process in `ps`.
 // Due to an obscure Mac bug, do not start this title with any symbol.
-process.title = 'ng';
+process.title = 'ng ' + Array.from(process.argv).slice(2).join(' ');
 
-var version = process.version.substr(1).split('.');
+const version = process.version.substr(1).split('.');
```

Comment:
This can potentially be run from an old version of node which doesn’t support const.  It would crash instead of displaying the error message

Reply:
Ah. I was wondering why that was there. Fixed.